scanner/src/main/kotlin/Main.kt

```diff
@@ -278,12 +283,18 @@ object Main {
         try {
             println("Scanning package '$identifier'...")
 
+            require(scanner is LocalScanner) {
```

Comment:
Why is this check already done here? Shouldn't it be only done in the `is File` case below?

Reply:
In general, `scanEntry()` is now only called with `input` being a `File` and thus can be simplified.